wiki/Available-Commands.md

```diff
@@ -0,0 +1,6 @@
+# Available Commands
```

Comment:
Aren't all of these commands currently documented in `Instruments.md`? We should drop this page until we have proper docs available.